crates/ruff_linter/src/rules/pylint/settings.rs

```diff
@@ -17,11 +17,11 @@ pub enum ConstantType {
 }
 
 impl ConstantType {
-    pub fn try_from_expr(expr: &Expr) -> Option<Self> {
-        match expr {
-            Expr::StringLiteral(_) => Some(Self::Str),
-            Expr::BytesLiteral(_) => Some(Self::Bytes),
-            Expr::NumberLiteral(ExprNumberLiteral { value, .. }) => match value {
+    pub fn try_from_literal_expr(literal_expr: LiteralExpressionRef<'_>) -> Option<Self> {
+        match literal_expr {
```

Comment:
Can we remove the catch al case or list the literals explicit to make the match exhaustive?

Reply:
Yeah, I'll list the literals explicitly.

I thought of merging multiple `ConstantType`, `LiteralType`, etc. into a single enum but it seems that one or the other variants are absent from them, so it's use-case dependent.